pkgs/objective_c/ffigen_objc.yaml

```diff
@@ -50,6 +50,27 @@ structs:
     - NSFastEnumerationState
   rename:
     _NSRange: NSRange
+enums:
```

Comment:
This and the interfaces, how do we determine the list? Hand-curated? Should we include everything? Why not?

(Possibly for a different PR though. Feel free to open an issue to track.)

Reply:
https://github.com/dart-lang/native/issues/1249